File: JavaScript/Conditionals/largestNumber.js
//Write a program that takes three numbers and prints the largest number.

let num1 = prompt("Enter number 1: ")
let num2 = prompt("Enter number 2: ")
let num3 = prompt("Enter number 3: ")


if (num1 >= num2 && num1 >= num3) {
    console.log("num1 is the largest")
}else if (num2 >= num1 && num2 >= num3){
    console.log("num2 is the largest")
}else{
    console.log("num3 is the largest")
}

File: Python/Conditionals/largestNumber.py
# Write a program that takes three numbers and prints the largest number.


num1 = int(input("enter first number: "))
num2 = int(input("enter second number: "))
num3 = int(input("enter third number: "))

if num1 >= num2 and num1 >= num3:
    print("The largest number is", num1)

elif num2 >= num3 and num2 >= num1:
    print("The largest number is ", num2)

else:
    print("The largest number is", num3)